packages/preview/CH4typ/0.0.1/typst.toml

```
@@ -0,0 +1,7 @@
+[package]
+name = "CH4typ"
+version = "0.0.1"
+license = "MIT"
+entrypoint = "lib.typ"
+authors = ["β-吲哚基丙氨酸"]
+description = "an alternative for chem-par packet"
```

In my opinion the description you wrote in your README is clearer to explain the purpose of the package, I would suggest copying it here.

```suggestion
description = "Typeset chemical formulas, both with inorganic and organic chemistry notation."
```